From 2a74a424ee588ae4df7505e6a667e9c14219b9b1 Mon Sep 17 00:00:00 2001
From: lilijun <lilijun@eswincomputing.com>
Date: Wed, 8 Jan 2025 16:12:18 +0800
Subject: [PATCH 377/551] WIN2030-16501:fix:Add vi config TBU

Changelogs:
The dw200 and the isp use a tbu.
When the dw200 switches the TBU of the VI,
the isp should also control it.

Change-Id: I647bd8f2b8ece766c7ca3ff27eccb4296ffd5146
Signed-off-by: lilijun <lilijun@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eswin-win2030-die0-soc.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/riscv/boot/dts/eswin/eswin-win2030-die0-soc.dtsi b/arch/riscv/boot/dts/eswin/eswin-win2030-die0-soc.dtsi
index ecfff2bf0334..cc651302b4a0 100644
--- a/arch/riscv/boot/dts/eswin/eswin-win2030-die0-soc.dtsi
+++ b/arch/riscv/boot/dts/eswin/eswin-win2030-die0-soc.dtsi
@@ -2055,6 +2055,7 @@ vi_top_csr: vi_common_top_csr@0x51030000 {
 			#size-cells = <2>;
 			reg = <0x0 0x51030000 0x0 0x10000>;
 			numa-node-id = <0>;
+			tbus = <WIN2030_TBUID_ISP>;
 		};
 
 
-- 
2.50.0

